test/plots/function-contour.js

```diff
@@ -0,0 +1,18 @@
+import * as Plot from "@observablehq/plot";
+
+export async function functionContour() {
+  return Plot.plot({
+    color: {
+      type: "diverging"
+    },
+    marks: [
+      Plot.contour({
+        value: (x, y) => x * y * Math.sin(x) * Math.cos(y),
```

Comment:
I feel like it would be more natural to specify **fill** here, rather than **value**. Maybe we can make that work as shorthand, but I didn’t want to support **fill** and **stroke** being set to _different_ functions (multiple overlapping contours?). 🤔

Reply:
yes it would be very nice; let's just throw an error if both are functions?

Reply:
I will work on more shorthand tomorrow.